### PR TITLE
refactor: added icons and refactor ui flow

### PR DESCRIPTION
**Pull request by ImmanuelSegol**
@peartobear Fixes you requested.

### CODE REVIEW

File: src/views/swap/steps.js [hunk 1]
```diff
@@ -53,7 +53,7 @@ class StyledStepOne extends React.Component {
     return (
       <View className={classes.wrapper}>
         <p className={classes.title}>
-          Paste a <b>Bitcoin</b> lightning <FaBolt size={30} color="#FFFF00" />{' '}
+          Paste a <b>Litecoin</b> lightning <FaBolt size={30} color="#FFFF00" />{' '}
```
Comment (michael1011) [inline src/views/swap/steps.js]: I think this should be `Bitcoin`

File: src/views/swap/steps.js [hunk 2]
```diff
@@ -139,7 +144,7 @@ const StyledStepTwo = ({ classes, value, address, link }) => (
           {value.sent} {value.sentCurrency}
         </b>{' '}
         <br />
-        on <b>Litecoin</b> <br />
+        on <b>Bitcoin</b> <br />
```
Comment (michael1011) [inline src/views/swap/steps.js]: And this `Litecoin`

File: src/scripts/utils.js [hunk 3]
```diff
@@ -7,3 +7,13 @@
...
 };
+
+/**
+ * Convert BTC to Satoshi.
+ *
+ * @param btc btc
+ * @returns satoshi
+ */
+export const toSatoshi = btc => {
+  return btc / 0.00000001;
```
Comment (michael1011) [inline src/scripts/utils.js]: What about:
```suggestion
  return btc * 100000000;
```

It is a little more elegant and I have had a rounding error when testing this PR.

**Comment by michael1011** [comment]
@ImmanuelSegol The denomination on this screen should also be changed to sats:

image

File: src/scripts/utils.js [hunk 4]
```diff
@@ -7,3 +7,13 @@
...
 };
+
+/**
+ * Convert BTC to Satoshi.
+ *
+ * @param btc btc
+ * @returns satoshi
+ */
+export const toSatoshi = btc => {
+  return (btc * 100000000 * 10) / 10;
```
Comment (michael1011) [inline src/scripts/utils.js]: Why `* 10) / 10`? Just multiplying the BTC amount times `100000000` should convert it to satoshis.

Comment (ImmanuelSegol) [reply src/scripts/utils.js]: Sorry didn't remove that I was testing something.

**Pull request merged**